lib/pages/movie_details_page.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/material.dart';
+import '../widgets/movie_detail_image.dart';
+import '../widgets/movie_detail_play_button.dart';
+import '../widgets/movie_details_actions.dart';
+import '../widgets/movie_details_info.dart';
+import '../widgets/movie_details_overview.dart';
+import '../widgets/movie_details_title.dart';
+import '../models/movie_result.dart';
+
+class MovieDetailPage extends StatelessWidget {
+  final MovieResult result;
+  final Function functionGetImage;
+
+  MovieDetailPage({
+    this.result,
+    this.functionGetImage,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Colors.black,
+      body: SingleChildScrollView(
+        child: Column(
+          children: [
+            MovieDetailImage(
+              image: functionGetImage(
+                result,
+              ),
+            ),
```

Comment:
Instead of send a widget, what do you think if you only have two parameter, the first one is for the posterPath and the second one relative to a default image path

Reply:
You can add another parameter like tag, like i see you use it in Hero widget

Reply:
Changed